recipes/entityx/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.32.0"
+
+
+class EntityXConan(ConanFile):
+    name = "entityx"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/alecthomas/entityx/"
+    topics = ("entity", "c++11", "type-safe", "component", "conan")
+    license = "MIT"
+    description = "EntityX is an EC system that uses C++11 features to provide type-safe component management, event delivery, etc."
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+    generators = "cmake"
+
+    exports_sources = "CMakeLists.txt", "patches/**"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("entityx-" + self.version, self._source_subfolder)
+
+    def validate(self):
+        if self.settings.compiler == "Visual Studio" and self.options.shared:
+            raise ConanInvalidConfiguration("entityx shared library does not export all symbols with Visual Studio")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["ENTITYX_BUILD_SHARED"] = self.options.shared
+        self._cmake.definitions["ENTITYX_BUILD_TESTING"] = False
+        self._cmake.definitions["ENTITYX_RUN_BENCHMARKS"] = False
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+    def build(self):
+        self._patch_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        self.copy("COPYING", src=self._source_subfolder, dst="licenses", keep_path=False)
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
```

Comment:
```suggestion
    def package_info(self):
        self.cpp_info.names["pkg_config"] = "entityx"
```

Last change, otherwise LGTM.
This is already the default value, but with this we enforce that we are official pkg_config name is `entityx`, so that others maintainers won't have to figure out if this is properly modeled or not: https://github.com/alecthomas/entityx/blob/82ab2d304ef5e6933cf6622721f086a1f4731e80/CMakeLists.txt#L194-L197

Reply:
Hi @SpaceIm and thanks for the many reviews!

I am interested in knowing a bit more about this one. Why doing it for ``pkg-config`` but not for others like ``cmake_find_package_multi``? Is it because the file would be expected to be called ``Entityx.pc`` by default in the pkg-config ecosystem?